tests/ast/nodes/test_evaluate_binop_int.py

```diff
@@ -0,0 +1,135 @@
+from hypothesis import (
+    example,
+    given,
+    settings,
+    strategies as st,
+)
+import pytest
+
+from vyper import (
+    ast as vy_ast,
+)
+from vyper.exceptions import (
+    ZeroDivisionException,
+)
+
+
+@pytest.mark.fuzzing
+@settings(deadline=500)
+@given(
+    left=st.integers(min_value=-(2 ** 32), max_value=2 ** 32),
+    right=st.integers(min_value=-(2 ** 32), max_value=2 ** 32),
+)
```

Comment:
`uint64_st` for below

```suggestion
int32_st = st.integers(min_value=-(2 ** 32), max_value=2 ** 32)
uint64_st = st.integers(min_value=0, max_value=2 ** 64)


@pytest.mark.fuzzing
@settings(deadline=500)
@given(left=int32_st, right=int32_st)
```